docker-compose.ci.yml

```
@@ -0,0 +1,9 @@
+services:
+  prod:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: [ "sh", "/app/scripts/build-out.sh" ]
+    volumes:
+      - .:/app
+      - ./out:/app/out
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)